src/process.rs

```diff
@@ -148,19 +148,25 @@ impl Process {
         }
     }
 
-    pub(crate) fn stdout(&self) -> Box<dyn filesource::Writer> {
+    pub(crate) fn stdout(&self) -> terminalsource::ColorableTerminal {
```

Comment:
Nit: still preferring importing instead of qualifying.

Reply:
I'm maintaning existing style within the code. If you want contributors to contribute in a different style, please change the style of the code.